docs/BreakingChanges/005 ListT.ForEach can throw exception when modifying list item.md

```diff
@@ -1,4 +1,4 @@
-## 5: `List<T>.ForEach`
+## 5: ListT.ForEach can throw exception when modifying list item
```

Comment:
ListT instead of List\<T\>? Unless you think the \\ to escape is too ugly in the raw mode, it might be nice to use \<T\> since it's more familiar syntax.